03-task-manager/starter/models/Task.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require("mongoose");
+
+// Only the properties set in your Schema will be passed unto the database.
+// Everything else will be ignored. When we use schema there will be a structure in our DB.
+// Model is a representation for the collection. So all the tasks to be pushed into our DB
+// will be added into that collection(model).
+const TaskSchema = new mongoose.Schema({
+    name: String,
+    completed:Boolean
+});
```

Comment:
This works -- and, you can define each property in the schema as an object that includes additional validation and defaults. [Take a look at the example to see how you can extend this.](https://github.com/duokobia/node-express-course/blob/main/03-task-manager/final/models/Task.js)

Reply:
Great.

Thank you for the link as well.

Reply:
Great.

Thank you for the link as well. I saw how John used it.